symengine/tests/test_functions.py

```diff
@@ -86,6 +90,11 @@ def test_Subs():
     assert Subs(Derivative(function_symbol("f", x, y), [x]), [x, y], [_x, x]) \
                 == Subs(Derivative(function_symbol("f", x, y), [x]), [y, x], [x, _x])
 
+    s = Subs(function_symbol("f", _x), [_x], [x])
```

Comment:
This is not a canonical expression. That's why there's a failure in Debug mode. I created https://github.com/symengine/symengine.py/issues/119 for this.

In the meantime, can you change this to `s = f.diff(x)/2` and change the asserts below accordingly?